pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -1492,10 +1493,45 @@ class Build {
                                                     file=$(basename "$f")
                                                     mv "$f" "${dir}/unsigned_${file}"
                                                     curl -o "$f" -F file="@${dir}/unsigned_${file}" -F entitlements="@$ENTITLEMENTS" https://cbi.eclipse.org/macos/codesign/sign
-                                                    chmod --reference="${dir}/unsigned_${file}" "$f"
-                                                    rm -rf "${dir}/unsigned_${file}"
+                                                    TESTMACSIGN=`grep -i "Apple Certification Authority" "$f"|wc -l`
+                                                    if [ $TESTMACSIGN -gt 0 ]
+                                                    then
+                                                      echo "Code Signed"
+                                                      chmod --reference="${dir}/unsigned_${file}" "$f"
+                                                      rm -rf "${dir}/unsigned_${file}"
+                                                    else
+                                                      max_iterations=20
+                                                      iteration=1
+                                                      while [ $iteration -le $max_iterations ]
+                                                      do
+                                                        echo "Code Not Signed - Have Another Try"
+                                                        sleep 1
+                                                        curl -o "$f" -F file="@${dir}/unsigned_${file}" -F entitlements="@$ENTITLEMENTS" https://cbi.eclipse.org/macos/codesign/sign
+                                                        TESTMACSIGN2=`grep -i "Apple Certification Authority" "$f"|wc -l`
```

Comment:
could extract that grep command into a const?

Reply:
Done!